app/controllers/paye/PayeGovernmentSpendController.scala

```diff
@@ -45,10 +45,13 @@ class PayeGovernmentSpendController @Inject()(
 
   def show(taxYear: Int): Action[AnyContent] = payeAuthAction.async { implicit request: PayeAuthenticatedRequest[_] =>
     payeAtsService.getPayeATSData(request.nino, taxYear).map {
+      case Right(_: PayeAtsData)
+          if (taxYear > appConfig.taxYear || taxYear < appConfig.taxYear - appConfig.maxTaxYearsTobeDisplayed) =>
+        Redirect(controllers.routes.ErrorController.authorisedNoAts(taxYear))
```

Comment:
That is not the response you used on the other pages? was is it not a forbidden response?

Reply:
Done